point.py

```diff
@@ -0,0 +1,135 @@
+import math  # I am guessing that you will need to use the math module
+import json  # I would like you to use the JSON module for reading geojson (for now)
+from math import sqrt
+from . import utils
+
+class Point():
+    def __init__(self, x=0, y=0, mark=[]):
+        self.x = x
+        self.y = y
+        self.mark = mark
+        
+    def read_geojson(self,input_file):

```

Comment:
Why is this method included in this class?  Does `read_geojson` have something to do with a point?  Below you utilize `utils.FUNCTION` notation that could also be used here.